bundle/manifests/operator-certification-operator.clusterserviceversion.yaml

```diff
@@ -215,13 +232,13 @@ spec:
         serviceAccountName: certification-operator-controller-manager
     strategy: deployment
   installModes:
-  - supported: false
+  - supported: true
```

Comment:
is this the right move?
from what i understand OLM is moving towards cluster singletons to avoid issues with conflicting installModes
https://github.com/operator-framework/operator-lifecycle-manager/issues/1790
https://groups.google.com/g/operator-framework/c/0KxBa-caG1U

Reply:
I feel like for this operator, since the resources we look for have to be in a specific namespace, and the ones we create are in that same namespace, this is the right approach for now. Maybe we change this once the install docs go away, and we enhance the operator further, but for now with how the operator is coded, and cluster admins have to apply a CatalogSource manually to get it in cluster, I think this is the right direction to be able to get a usable alpha release out.